.github/workflows/release.yaml

```diff
@@ -0,0 +1,141 @@
+name: build
+
+on:
+  push:
+    tags:
+      - "*"
+jobs:
+  mac:
+    strategy:
+      fail-fast: false
+      matrix:
+        otp:
+          - 24.2.1-1
+          - 23.3.4.9-3
+        os:
+          - macos-11
+          - macos-10.15
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: prepare
+        run: |
+          brew update
+          brew install curl zip unzip gnu-sed kerl
+          echo "/usr/local/bin" >> $GITHUB_PATH
+          git config --global credential.helper store
+      - uses: actions/cache@v2
+        id: cache
+        with:
+          path: ~/.kerl/${{ matrix.otp }}
+          key: otp-install-${{ matrix.otp }}-${{ matrix.os }}
+
+      - name: build erlang
+        if: steps.cache.outputs.cache-hit != 'true'
+        timeout-minutes: 60
+        run: |
+          kerl update releases
+          kerl build git https://github.com/emqx/otp.git OTP-${{ matrix.otp }} ${{ matrix.otp }}
+          kerl install ${{ matrix.otp }} $HOME/.kerl/${{ matrix.otp }}
+
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: install rebar3
+        run: |
+          wget https://s3.amazonaws.com/rebar3/rebar3 && chmod +x rebar3
+          cp ./rebar3 /usr/local/bin/rebar3
+
+      - name: build release
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          . $HOME/.kerl/${{ matrix.otp }}/activate
+          BUILD_RELEASE=1 make
+
+      - uses: actions/upload-artifact@v2
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          name: packages
+          path: |
+            _packages/*.gz
+            _packages/*.gz.sha256
+
+  linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        otp:
+          - 23.3.4.9-4
+          - 24.2.1-1
+        elixir:
+          - 1.13.3
+        arch:
+          - amd64
+          - arm64
+        os:
+          - ubuntu20.04
+          - ubuntu18.04
+          - ubuntu16.04
```

Comment:
remove ubuntu 16.04?

Reply:
we have the builder.
if there is an issue with the builder, we can have it removed.